ledfx/effects/audio.py

```
@@ -47,6 +56,81 @@ class AudioInputSource:
+            try:
+                if hasattr(stream, "abort"):
+                    stream.abort()
+                    _LOGGER.info("%sStream aborted successfully", prefix)
+                    stop_ok = True
+                else:
+                    _LOGGER.warning(
+                        "%sStream has no abort() method; "
+                        "proceeding to close()",
+                        prefix,
+                    )
+            except Exception as e:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)